modules/apps/layout/layout-page-template-service/src/main/java/com/liferay/layout/page/template/service/impl/LayoutPageTemplateEntryServiceImpl.java

```diff
@@ -609,6 +609,24 @@ public int getLayoutPageTemplateEntriesCountByType(
 			groupId, layoutPageTemplateCollectionId, type);
 	}
 
+	@Override
+	public LayoutPageTemplateEntry getLayoutPageTemplateEntry(
+			long groupId, String layoutPageTemplateEntryKey)
+		throws PortalException {
+
+		LayoutPageTemplateEntry layoutPageTemplateEntry =
+			layoutPageTemplateEntryLocalService.getLayoutPageTemplateEntry(
+				groupId, layoutPageTemplateEntryKey);
+
+		if (layoutPageTemplateEntry != null) {
```

Comment:
I think this can't be null because the findBy... would have thrown an exception if the entry is not found in the database